### ISSUE TITLE
Better bug report

### ISSUE THREAD

**Issue by bakeiro** [Couchers-org/couchers#5266]
[needs design] 

## Context ✏️:
I was working with Google cloud when suddenly a bug occur, automaticaly a pop-up shows up so I can write what I was doing before the bug to occur, and allows me to take a screenshot and highlight the relevant info, so the person who sees the bug has way more info!

this ticket a research ticket, my idea it's to show this feature, then a UI/UX designer could adapt this feature in our site (using our wording, UI style, etc)

## Screenshots 📸:
![image](https://github.com/user-attachments/assets/b567400e-d79a-471c-8cc3-f41e74964544)
![image](https://github.com/user-attachments/assets/f22754f2-a723-4add-ae7b-d4af9a0ae476)

**Comment by jesseallhands** [comment]
We could use this feature since we already use sentry: https://sentry.io/for/user-feedback/

**Comment by bakeiro** [comment]
> We could use this feature since we already use sentry: https://sentry.io/for/user-feedback/

That's true... we could even record user interactions when a crash occurs... and automatically, i think it's better...

**Comment by aapeliv** [comment]
Superseded by https://github.com/Couchers-org/couchers/issues/5291
